src/os2datascanner/projects/admin/adminapp/views/miniscanner_views.py

```diff
@@ -84,10 +79,31 @@ def execute_mini_scan(request):  # noqa:CCR001
                         if not message.matched:
                             continue
 
-                        replies.append(message)
+                        yield message
             else:
                 logger.warning(
                         "Miniscanner -"
                         " Rejected file that exceeded the size limit.")
```

Comment:
Can this also occur if the text is too big? If so, you could consider making this warning more accurate.

Reply:
The file size limit is set to 1Mb, and pasting 1Mb into the text field actually crashes the page. Adding a big amount will cause a 400 bad request, so the easiest fix would be (imo) to check if a character limit is possible within HTML, or even using some javascript that checks the text field size before sending anything. Let me know if you think there is an easier way.

Reply:
It looks like `input` has a parameter `maxlength`, which I think would be the easiest.

Reply:
Yes ! It works great, thank you, and is definitely easier than calling js functions and making the codebase even bigger.